server/historian/packages/historian-base/src/utils.ts

```diff
@@ -32,6 +32,13 @@ export function normalizePort(val) {
 	return false;
 }
 
+export const Constants = Object.freeze({
+	historianRestThrottleIdSuffix: "HistorianRest",
+	createSummaryThrottleIdPrefix: "createSummary",
+	getSummaryThrottleIdPrefix: "getSummary",
+    generalRestCallThrottleIdPrefix: "generalRestCall",
```

Comment:
nit: tabs/spacing looks a bit off. Can you run `pnpm run prettier:fix` in server/historian?

Reply:
Done.